frontend/themes/ce-demo/components/vaadin-dialog-overlay.css

```diff
@@ -1,7 +1,19 @@
 :host([theme~="no-padding"]) [part="content"] {
   padding: 0;
 }
-/* TODO: TEMPORARY FIX UNTIL https://github.com/vaadin/collaboration-engine-internal/pull/548 IS MERGED AND RELEASED */
-[part="overlay"] {
+
+:host([theme~="editor-view-dialog"]) [part="overlay"] {
+  height: 100%;
+  max-width: 400px;
+  width: 100%;
+  /* Temporary fix https://github.com/vaadin/collaboration-engine-internal/pull/548 */
   z-index: -1;
```

Comment:
The fix is already included in the used platform version, so this workaround could be removed.

Reply:
Done.